src/styles/measures.scss

```diff
@@ -0,0 +1,5 @@
+$width-card: 200px;
+$row-gap: 46px;
+$column-gap: 48px;
+$paddingH-card: 40px;
```

Comment:
Variable names should be written in kebab-case. Please rename `$paddingH-card` to `$padding-h-card` to adhere to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).